isso/ext/notifications.py

```diff
@@ -76,7 +76,14 @@ def __init__(self, isso):
 
         self.isso = isso
         self.conf = isso.conf.section("smtp")
-        self.public_endpoint = isso.conf.get("server", "public-endpoint") or local("host")
+        self.public_endpoint = isso.conf.get("server", "public-endpoint")
+        # rstrips potential trailing '/', without having to `repr` the `local` object.
```

Comment:
I don't understand this comment - what does repr have to do with it? Why is it bad to strip / from the output of local("host") ?

Reply:
Please see #523. The first patch (https://github.com/posativ/isso/commit/1a3f803b966cd32da367978f306b923d433c5cda) dealt with the output of `local("host")` as a string and hard-failed **at this step**.
That's why the r-strip is only applied on an URL fetched from configuration.